models/product-selection/src/product-selection-draft/builder.ts

```diff
@@ -0,0 +1,15 @@
+import { Builder } from '@commercetools-test-data/core';
+import type {
+  TCreateProductSelectionDraftBuilder,
+  TProductSelectionDraft,
+} from '../types';
+import generator from './generator';
+import transformers from './transformers';
+
+const Model: TCreateProductSelectionDraftBuilder = () =>
```

Comment:
As I learn more about the test-data repo, is there a readme somewhere that explains the roles and responsibilities of the different terms here: Model, builder, generator, transformer? Or is this just standard knowledge for a specific pattern?

Reply:
Should I contact your people or you contact my people for us to discuss? Let's set aside an hour to let me absorb all the info.